addons/website_slides/controllers/main.py

```diff
@@ -1077,7 +1077,7 @@ def create_slide(self, *args, **post):
 
         if post.get('duration'):
             # minutes to hours conversion
-            values['completion_time'] = int(post['duration']) / 60
+            values['completion_time'] = int(float(post['duration'])) / 60
```

Comment:
It makes sense to only keep integers as only integers will be shown too.
I suggest however the following:

1. Set a minimum value of `0`
```suggestion
            values['completion_time'] = max(int(float(post['duration']))/ 60, 0)
```
2. Modify the template(s) to reflect this constraint (add `step=1` and `min=0` on the `input`(s)).